### DISCUSSION TITLE
LOW: Permissions required to jump the queue

### DISCUSSION THREAD

**Discussion by purajit** [community/community#65496] [Pull Requests]
Considering this a bug since it isn't documented, as far as I can tell.

Only repo admins have the ability to jump the queue, but this isn't mentioned anywhere. It is also an odd choice, since users with `Write` access can still remove PRs from the queue, and can remove their own PR, update their own PR forcibly, etc, which all have the same effect of impacting the rest of the queue. What makes jumping the queue any different?

There is no mention of any additional permissions required in the documentation [here](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-a-merge-queue#jumping-to-the-top-of-the-queue) or [here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/merging-a-pull-request-with-a-merge-queue), and even in [this answer](https://github.com/orgs/community/discussions/46757#discussioncomment-4909755) by a Github contributor, the implication is that everyone should be able to jump the queue:

> Jumping is useful if you have a pull request that needs to get merged ASAP, but you want it to go through your normal workflow and pass all required status checks. Alternatively, an admin (depending on protected branch settings) can bypass the queue and merge a pull request directly even if it has failing status checks. This

I'm currently working around this by creating a custom role, but I'd like to not have to do that. At the very least, it would be nice to:
* have this documented, and reflected in the answers given (the phrasing of the answers and docs makes me think this isn't entirely intentional)
* have a reasoning for why jumping is any different than the other actions that can impact the queue in the same way

**Comment by miluoshi** [comment]
This still doesn't seem to be addressed in the documentation. Any updates?

**Comment by TTMichaelA** [comment]
I'd also like an update on this. For obvious reasons, this is problematic, especially if you have enabled merge queues to get around the issue of having to repeatedly update branches with long CI checks.
